samples/nextjs/src/components/Styleguide-Layout/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Placeholder, getChildPlaceholder, getFieldValue } from '@sitecore-jss/sitecore-jss-nextjs';
+
+/**
+ * The main layout (columns) of the styleguide.
+ * Navigation is automatically generated based on the components added to the layout,
+ * and does not need to be manually maintained.
+ */
+const StyleguideLayout = (props) => {
+  // this code reads the components in the child placeholders of this component,
+  // and projects them into the left navigation column for the styleguide
+  const sections = getChildPlaceholder(props.rendering, 'jss-styleguide-layout')
+    .filter((section) => getFieldValue(section, 'heading'))
+    .map((section) => ({
+      heading: getFieldValue(section, 'heading'),
+      id: `i${section.uid.replace(/[{}]/g, '')}`,
+      children: getChildPlaceholder(section, 'jss-styleguide-section')
+        .filter((component) => getFieldValue(component, 'heading'))
+        .map((component) => ({
+          heading: getFieldValue(component, 'heading'),
+          id: `i${component.uid.replace(/[{}]/g, '')}`,
+        })),
+    }))
+    .map((section) => (
+      <nav key={section.heading} className="nav flex-column pt-2">
+        <a href={`#${section.id}`} className="nav-item font-weight-bold">
+          {section.heading}
+        </a>
+        {section.children && (
+          <nav className="nav flex-column">
+            {section.children.map(
+              (child) =>
+                child.heading && (
+                  <a key={child.id} href={`#${child.id}`}>
```

Comment:
@ambrauer Should we wrap links with anchor by `next/Link`?

Reply:
Same here - these aren't links between pages/routes, just navigation within the page.